### PR TITLE
Sign_Up Page for Ambu

### PR DESCRIPTION
**Pull request by Pal-prakash**
In this PR, we have made the sign up page for ambu as per the design iterations passed to the tech team. Below are the proof of changes on both mobile device and desktop.

## Proof that changes are correct on Desktop

https://github.com/AMBU-The-Unified-HealthCare-System/Ambu-Web-App/assets/120311694/d4cb4f48-991e-43a2-9317-4dc3dbfd3c7d

## Proof that changes are correct on Mobile 

https://github.com/AMBU-The-Unified-HealthCare-System/Ambu-Web-App/assets/120311694/849e9113-5fcd-461c-988b-2820afee2f27

### CODE REVIEW

**Comment by Ash-2k3** [comment]
Thanks @Pal-prakash, can you also add a screen-recording of mobile view (smaller screens). I will start review once you have added it.

**Comment by Pal-prakash** [comment]
@Ash-2k3 Sir I added the mobile view.
https://github.com/AMBU-The-Unified-HealthCare-System/Ambu-Web-App/assets/120311694/849e9113-5fcd-461c-988b-2820afee2f27

**Comment by Ash-2k3** [comment]
Thanks @Pal-prakash.
 Deferring to @Yash080902 for design and responsiveness approval, If it looks good to him. I will start reviewing code.

**Review by Ash-2k3 — CHANGES_REQUESTED** [review]
Left some comments, PTAL

File: app/__init__.py [hunk 1]
```diff
@@ -8,5 +8,9 @@ def create_app():
     
     # Register the blueprints for endpoints in app instance
     app.register_blueprint(controllers.get_index_bp)
+    app.register_blueprint(controllers.get_Sign_Up)
+    
+    return app
```
Comment (Ash-2k3) [inline app/__init__.py:12]: Please get rid of whitespaces and blanks lines here and elsewhere too. Please follow this for all the files and tag me once it's done, Thanks

File: app/controller/controllers.py [hunk 2]
```diff
@@ -9,4 +9,9 @@
 @get_index_bp.route('/')
 def index():
            # Render index.html
-           return render_template('index.html')
\ No newline at end of file
+           return render_template('index.html')
+
+@get_Sign_Up.route('/Sign_Up')
```
Comment (Ash-2k3) [inline app/controller/controllers.py:14]: Are you sure it's working ? I can't see `get_Sign_up` declared. You can take reference of `get_index_bp` above in line 7

**Review by Ash-2k3 — CHANGES_REQUESTED** [review]
Thanks, left some comments, PTAL.

**Comment by Pal-prakash** [comment]
@Ash-2k3 i made changes. Sorry for missing it

**Comment by Ash-2k3** [comment]
Thanks @Pal-prakash. Also just a note: Please don't resolve the comments by yourself, let the reviewers do it once they are satisified wid your changes. This helps in keeping track of the changes they asked :)

File: app/controller/controllers.py (hunk 2 above)
Comment (Ash-2k3) [reply app/controller/controllers.py:14]: No, this is not quite correct, please declare the variables above, after line 7.

File: app/templates/Sign_Up.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="..\static\css\Sign_Up.css">
+    <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>                       
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Jaldi:wght@700&family=Kadwa:wght@700&display=swap" rel="stylesheet">
+    <title>Sign Up</title>
+</head>
+<body>
+
```
Comment (Ash-2k3) [inline app/templates/Sign_Up.html]: Nit: Remove the whitespace

File: app/templates/Sign_Up.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    <div id="header">
+
```
Comment (Ash-2k3) [inline app/templates/Sign_Up.html]: Same here, remove the whitespace

**Review by Ash-2k3 — CHANGES_REQUESTED** [review]
Thanks, left some minor comments. Otherwise this looks good and can get merged once you have finished addressing these minor comments

**Comment by Pal-prakash** [comment]
@Ash-2k3 Sir, I removed all whitespaces

**Comment by Ash-2k3** [comment]
Thanks @Pal-prakash, LGTM !!

**Review by Ash-2k3 — APPROVED** [review]

**Pull request merged**